src/main/java/seedu/address/logic/commands/RemoveGroupCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GROUP;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.group.Group;
+import seedu.address.model.person.exceptions.GroupCannotBeRemovedException;
```

Comment:
I think it's better if we put the Group Exceptions int the seedu.address.model.group.exceptions package like the TagNotFoundException in seedu.address.model.tag.exceptions

Reply:
got it 👍